src/sc/sidechaintypes.h

```diff
@@ -483,6 +483,26 @@ typedef struct sScFeeData_tag
 
 } ScFeeData;
 
+struct ScFeeData_v2 : public ScFeeData {
+    int submissionHeight;
+    ScFeeData_v2() : ScFeeData(), submissionHeight(-1) {};
+    ScFeeData_v2(CAmount f, CAmount m, int submHeight) : ScFeeData(f, m), submissionHeight(submHeight) {};
+
+    ADD_SERIALIZE_METHODS;
+
+    template <typename Stream, typename Operation>
+    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int nVersion) {
+        READWRITE(forwardTxScFee);
+        READWRITE(mbtrTxScFee);
+        READWRITE(submissionHeight);
+    }
+
+    inline bool operator==(const ScFeeData_v2& rhs) const
+    {
+        return (forwardTxScFee == rhs.forwardTxScFee && mbtrTxScFee == rhs.mbtrTxScFee && submissionHeight == rhs.submissionHeight);
```

Comment:
Similar to the comment above, how about
```suggestion
        return ScFeeData::operator=(rhs) && submissionHeight == rhs.submissionHeight;
```

Reply:
Thanks, I updated both function as suggested in cb80200cd4274e56654dd8b0860c37dcb80cc076